packages/web/src/pages/settings-page/components/desktop/DeveloperApps/DeveloperAppsSettingsModal.tsx

```diff
@@ -0,0 +1,83 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import {
+  Modal,
+  ModalHeader,
+  ModalProps,
+  ModalTitle,
+  ModalContentPages
+} from '@audius/stems'
+
+import { ReactComponent as IconEmbed } from 'assets/img/iconEmbed.svg'
+
+import { AppDetailsPage } from './AppDetailsPage'
+import { CreateNewAppPage } from './CreateNewAppPage'
+import { YourAppsPage } from './YourAppsPage'
+import { CreateAppsPages } from './types'
+
+const messages = {
+  title: 'Your Apps',
```

Comment:
Looks like these are intentionally the same as the ones in YourAppsPage so prob better to share the same `messages` obj across the two files